services/madoc-ts/src/capture-model-server/server-filters/remove-fields-deleted-by-published-revisions.ts

```diff
@@ -0,0 +1,6 @@
+import { CaptureModel } from '../../frontend/shared/capture-models/types/capture-model';
+
+export function removeFieldsDeletedByPublishedRevisions(model: CaptureModel): CaptureModel {
+  // @todo implement fields deleted by published revisions.
```

Comment:
Need to create ticket to cover this and the implications (if any) I can't remember.